src/Sulu/Bundle/ContentBundle/Resources/config/document.xml

```diff
@@ -92,14 +92,19 @@
             <tag name="sulu_document_manager.event_subscriber" />
         </service>
 
-        <service id="sulu_document_manager.document.subscriber.route"
-                 class="Sulu\Component\Content\Document\Subscriber\RouteSubscriber">
+        <service id="sulu_document_manager.document.subscriber.route_generating"
+                 class="Sulu\Component\Content\Document\Subscriber\RouteGeneratingSubscriber">
             <argument type="service" id="sulu_document_manager.document_manager"/>
             <argument type="service" id="sulu_document_manager.document_inspector"/>
             <argument type="service" id="sulu.phpcr.session"/>
             <tag name="sulu_document_manager.event_subscriber" manager="default" />
```

Comment:
This has now become somewhat pointless - we need also to disable this subscriber when syncing FROM the Target DM to the "default" DM - which are we doing quite awkardly with an option.